src/tests/Microsoft.PowerFx.Core.Tests/ResourceValidationTests.cs

```diff
@@ -17,37 +17,14 @@ public class ResourceValidationTests : PowerFxTest
         [Fact]
         public void ResourceLoadsOnlyRequiredLocales()
         {
-            // Get a string from En-Us to ensure it's loaded
+            // Get a string from En-Us and to De-DE ensure they're loaded
             Assert.NotNull(StringResources.Get("AboutIf", "en-US"));
+            Assert.NotNull(StringResources.Get("ErrGeneralError", "de-DE"));
 
-            var loaded = string.Empty;
-            var loadedCount = 0;
+            var assemblies = AppDomain.CurrentDomain.GetAssemblies();
 
-            void ResourceAssemblyLoadHandler(object sender, AssemblyLoadEventArgs args)
-            {
-                loaded = args.LoadedAssembly.FullName;
-                loadedCount++;
-            }
-
-            try 
-            {
-                AppDomain.CurrentDomain.AssemblyLoad += ResourceAssemblyLoadHandler;
```

Comment:
> AppDomain.CurrentDomain.AssemblyLoad += ResourceAssemblyLoadHandler; [](http://example.com/codeflow?start=15&length=69)

discusse offline:
We remove this because it's a race condition. 
Other tests may run concurrently and load assemblies, so this test would randomly fail.